src/OrderPage/index.js

```diff
@@ -0,0 +1,25 @@
+import './styles.scss';
+import {useEffect, useState} from 'react';
+import Hero from '../Hero';
+import Restaurant from '../Restaurant';
+const OrderPage = ({ restaurantId }) => {
+
+    const [menu, setMenu] = useState([])
+
+    const getMenu = () => {
+        fetch(`http://localhost:8080/restaurants/${restaurantId}`)
+            .then(data => data.json())
+            .then(menu => {
+                setMenu(menu)
+            })
+    }
+
+    useEffect(() => {
+        getMenu()
+    }, [menu])
```

Comment:
Should be [restaurantId] not [menu] as we want it to re-render when the restuarant is changed.